packages/console/src/hooks/use-swr-options.ts

```diff
@@ -4,7 +4,7 @@ import useApi, { RequestError } from './use-api';
 import useSwrFetcher from './use-swr-fetcher';
 
 const useSwrOptions = (): SWRConfiguration => {
-  const api = useApi({ hideErrorToast: true });
+  const api = useApi();
```

Comment:
This will cause infinite error toasts flushing on your screen. Are you sure you want to remove it?

Reply:
it won't i think? per slack discussion

Reply:
Oh, if there's no more "404 retry" then I'm good with it. Just realized that 401 and 403 will not cause swr retry in our config